src/SDKs/ContainerInstance/Management.ContainerInstance/Microsoft.Azure.Management.ContainerInstance.csproj

```diff
@@ -5,7 +5,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure Container Instance Management Library</Description>
-		<Version>1.0.1-preview</Version>
+		<Version>1.0.2-preview</Version>
```

Comment:
Missing < PackageReleaseNotes >. Add release notes as to what is being fixed/added to this release.

Reply:
@shahabhijeet , fixed. Please take a look.